docs/section9/Rule6-jl.md

```diff
@@ -2,38 +2,55 @@
 
-    - If the space_type is "Guest Room", or "Dormitory Living Quarters", or "Dwelling Unit": ```if space_type == "Guest Room" or space_type == "Dormitory Living Quarters" or space_type == "Dwelling Unit"```  
+    - If the lighting_space_type is "Guest Room", or "Dormitory Living Quarters": ```if ( lighting_space_type_proposed == "Guest Room" or "Dormitory Living Quarters" ):```  
 
-      - Get the lighting power denstiy allowance: ```lighting_power_allowance = data_lookup(table_9_6_1, space_type)```
+      - Get the lighting power density allowance for the proposed model: ```lighting_power_allowance_proposed = data_lookup(table_9_6_1, lighting_space_type_proposed)```  
 
-      - Get the total lighting power modeled in the space: ```space_lighting_power_proposed = space.lighting_power```  
+    - If the lighting_space_type is "Dwelling Unit": ```else: lighting_power_allowance_proposed = 0.6```  
 
-      - Get the area of the space: ```floor_area_proposed = space.floor_area```  
+    - Get interior_lighting in space: ```interior_lighting_proposed = space_proposed.interior_lightings```  
 
-      **Rule Assertion:** For each space: ```space_lighting_power_proposed <= floor_area_proposed * lighting_power_allowance```  
+      - Get the total design power_per_area for the space: ```space_lighting_power_per_area_proposed = sum( lighting.power_per_area for lighting in interior_lighting_proposed )```  
+
+    - Get the matching space in U_RMR: ```space_user = match_data_element(U_RMR, spaces, space_proposed.name)```  
+
+      - Get the total design power_per_area for the space: ```space_lighting_power_per_area_user = sum( lighting.power_per_area for lighting in space_user.interior_lightings )```  
+
+        **Rule Assertion:** For each space that is Hotel/Motel Guestroom or Dormitory Living Quarters, or Dwelling Units in the proposed model: ```space_lighting_power_per_area_proposed = max( lighting_power_allowance_proposed, space_lighting_power_per_area_user)```  
+
+    - Get matching space from Baseline RMR: ```space_baseline = match_data_element(B_RMR, spaces, space_proposed.name)```  
+
+      - Get the lighting power density allowance for the baseline model: ```lighting_power_allowance_baseline = data_lookup(table_G_3_7, lighting_space_type_proposed)```  
+
+      - Get interior_lighting in space: ```interior_lighting_baseline = space_baseline.interior_lightings```  
+
+      - Get the total baseline power_per_area for the space: ```space_lighting_power_per_area_baseline = sum( lighting.power_per_area for lighting in interior_lighting_baseline )```  
+
+        **Rule Assertion:** For each space that is Hotel/Motel Guestroom or Dormitory Living Quarters, or Dwelling Units in the baseline model: ```space_lighting_power_per_area_baseline == lighting_power_allowance_baseline```  
```

Comment:
@xingcx this looks perfect now! My only comment would be to split this into 2 RDS- one for the proposed and one for the baseline. This is because this rule has two rule assertions and we would like to have only one rule assertion for each RDS. You can just call the second one 6-jl-2. we will renumber all RDSs eventually.

Reply:
I agree

Reply:
@mkarpman I agree. There will also be a separate RDS to check for controls in baseline and another one for proposed. (see the lighting TCDs).
@xingcx - the only change this one needs is to be sl=plit into 2 RDS. One for proposed for these specific space types and one for baselines for all space types.